src/data/gqlQueries.js

```diff
@@ -66,6 +66,7 @@ export async function updateRequest(request) {
     mutation {
       updateRequest(input: {
```

Comment:
```suggestion
      request:updateRequest(input: {
```
And change the return statement to `return response.request || {}` (it's too far out of scope for the suggest changes feature)